pkg/hubble/monitor/consumer.go

```diff
@@ -77,13 +88,23 @@ func (c *consumer) sendEvent(event *observerTypes.MonitorEvent) {
 	if c.numEventsLost > 0 {
 		c.sendNumLostEvents()
 	}
-
 	select {
 	case c.observer.GetEventsChannel() <- event:
 	default:
-		c.observer.GetLogger().Debug("hubble events queue is full, dropping message")
-		c.numEventsLost++
+		c.logStartedDropping()
```

Comment:
this is a functional change, too, not just a bug.

we used to log in debug for each message and now only do it once. Either this is not an intentional change, or we should mention it as a minor note in the release